server/routes/profile.js

```diff
@@ -0,0 +1,11 @@
+const  express= require('express');
+const router = express.Router();
+const multer = require('multer');
+const upload = multer({ 'dest': 'uploads/' })
+
+const { createProfile } = require("../controllers/profile");
+
+
+router.route('/').post( upload.array('images',4),createProfile);
```

Comment:
:+1:

Reply:
Altho, I changed it to upload.single() for now.